packages/radio-group/src/Radio.tsx

```diff
@@ -1,76 +1,216 @@
+  font-size: 14px;
+  line-height: 20px;
 `;
 
+// Note colors are not in our palette
+const inputVariantStyle = {
+  [Variant.Default]: css`
+    &:checked + ${styledDiv.selector} {
+      background-color: #2798bd;
+      border-color: #2798bd;
+
+      &:after {
+        transform: scale(1);
+      }
+    }
+
+    &:hover + ${styledDiv.selector}:before {
+      border: 3px solid rgba(184, 196, 194, 0.3);
+    }
+
+    &:focus + ${styledDiv.selector}:before {
+      border: 3px solid #9dd0e7;
+    }
+
+    &:disabled + ${styledDiv.selector} {
+      border-color: ${uiColors.gray.light1};
+      background-color: ${uiColors.gray.light2};
+    }
+  `,
+
+  [Variant.Light]: css`
+    &:checked + ${styledDiv.selector} {
+      background-color: #43b1e5;
+      border-color: #43b1e5;
+
+      &:after {
+        transform: scale(1);
+      }
+    }
+
+    &:hover + ${styledDiv.selector}:before {
+      border: 3px solid rgba(255, 255, 255, 0.2);
+    }
+
+    &:focus + ${styledDiv.selector}:before {
+      border: 3px solid ${uiColors.blue.base};
+    }
+
+    &:disabled + ${styledDiv.selector} {
+      background-color: ${uiColors.gray.dark2};
+      border-color: rgba(255, 255, 255, 0.15);
+    }
+  `,
+};
+
+const disabledChecked = {
+  [Variant.Default]: css`
+    &:disabled + ${styledDiv.selector} {
+      background-color: ${uiColors.gray.light1};
+      border-color: ${uiColors.gray.light1};
+
+      &:after {
+        background-color: ${uiColors.gray.light2};
+        transform: scale(1);
+      }
+    }
+  `,
+
+  [Variant.Light]: css`
+    &:disabled + ${styledDiv.selector} {
+      border-color: ${uiColors.gray.dark2};
+
+      &:after {
+        background-color: rgba(255, 255, 255, 0.3);
+        transform: scale(1);
+      }
+    }
+  `,
+};
+
 const inputStyle = css`
-  margin-right: 0.5em;
-  cursor: pointer;
+  height: 0;
+  width: 0;
+  opacity: 0;
 
-  &:disabled {
+  &:disabled:hover + ${styledDiv.selector}:before {
+    border: none;
+  }
+
+  &:disabled + ${styledDiv.selector} {
     cursor: not-allowed;
+
+    :after {
+      box-shadow: none;
+    }
   }
 `;
 
-const disabledButtonVariant = {
-  default: css`
-    color: ${colors.gray[5]};
-    cursor: not-allowed;
+const divVariantStyle = {
+  [Variant.Default]: css`
+    border: 2px solid ${uiColors.gray.dark1};
+    background-color: rgba(255, 255, 255, 0.31);
   `,
 
-  light: css`
-    color: ${colors.gray[4]};
-    cursor: not-allowed;
+  [Variant.Light]: css`
+    border: 2px solid ${uiColors.white};
+    background-color: rgba(255, 255, 255, 0.2);
   `,
 };
 
-const textStyle = css`
-  line-height: 0.9em;
+const divStyle = css`
+  position: relative;
+  transition: background-color 0.15s ease-in-out;
+  background-color: white;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  flex-shrink: 0;
+  border-radius: 100px;
+  margin-right: 8px;
+
+  &:before {
+    content: '';
+    position: absolute;
+    border-radius: 100px;
```

Comment:
When creating circles, we should use `100%`.

Reply:
👍